cucoslib/defaultconf.py

```diff
@@ -9,8 +9,8 @@
     # URL to npmjs couch DB, which returns stream of changes happening in npm registry
     "npmjs_changes_url": "https://skimdb.npmjs.com/registry/_changes?descending=true&include_docs=true&feed=continuous",
     "git": {
-        "user_name": "cucos",             # git config user.name
-        "user_email": "cucos@cucos.ccs"   # git config user.email
+        "user_name": "f8a",             # git config user.name
+        "user_email": "f8a@f8a.ccs"   # git config user.email
```

Comment:
The `.ccs` suffix was meant as abbreviation from cucos, so I'd go with *f8a@f8a.f8a* or just *f8a@f8a* (`git config --global user.email` doesn't check the value and it's nowhere used so it could be even *f8a*)